### PR TITLE
Fix build issue with openshift API version in go.mod

### PR DESCRIPTION
**Pull request by dmvolod**
**Release Note**
This changes in go.mod fix `github.com/openshift/api@v3.9.1-0.20190927182313-d4a64ec2cbd8+incompatible: invalid pseudo-version: preceding tag (v3.9.0) not found` on download and build go modules
```release-note
NONE
```

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -61,6 +61,8 @@ replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503
 // Using a fork that removes the https ping before using http in case of insecure registry (for Spectrum)
 replace github.com/google/go-containerregistry => github.com/nicolaferraro/go-containerregistry v0.0.0-20200428072705-e7aced86aca8
 
+replace github.com/openshift/api => github.com/openshift/api v0.0.0-20190927182313-d4a64ec2cbd8
```
[the remark this answers is not in the document]
Comment (astefanutti) [reply go.mod:64]: Could you also add a comment that mentions the OpenShift version that commit relates?

[the remark this answers is not in the document]
Comment (astefanutti) [reply go.mod:64]: @dmvolod I mean changing:

```
github.com/openshift/api v3.9.1-0.20190927182313-d4a64ec2cbd8+incompatible
```

directly with:

```
github.com/openshift/api v0.0.0-20190927182313-d4a64ec2cbd8
```

[the remark this answers is not in the document]
Comment (dmvolod) [reply go.mod:64]: Oh, thanks, will try to change as you suggested

[the remark this answers is not in the document]
Comment (dmvolod) [reply go.mod:64]: @astefanutti it seams to be not compiled anymore after recommended changes. Let's keep current solution.

**Comment by dmvolod** [comment]
@nicolaferraro , @lburgazzoli could you please a look, seems to be compile from scratch (without initial vendor folder) cause this problem

**Comment by dmvolod** [comment]
Closing as no more feedback for a long time

**Pull request closed without merging**